docs/src/pages/[platform]/components/heading/demo.tsx

```diff
@@ -1,21 +1,44 @@
 import * as React from 'react';
-import { Heading, HeadingLevel } from '@aws-amplify/ui-react';
+import { Heading, HeadingLevel, HeadingProps } from '@aws-amplify/ui-react';
 
 import { HeadingPropControls } from './HeadingPropControls';
 import { Demo } from '@/components/Demo';
 
-const propsToCode = (level: HeadingLevel) => {
-  return `<Heading level={${level}}>Heading</Heading>`;
+const propsToCode = (
+  level: HeadingLevel,
+  isTruncated: boolean,
+  value: string
+) => {
+  return `<Heading
+width='30vw'
+level={${level}} 
+isTruncated={${isTruncated}}>
+  ${value}
+</Heading>`;
 };
 export const HeadingDemo = () => {
   const [level, setLevel] = React.useState<HeadingLevel>(6);
-
+  const [isTruncated, setIsTruncated] = React.useState<boolean>(false);
+  const [value, setValue] = React.useState<string>('Hello World!');
+  console.log('isTruncated', isTruncated);
   return (
     <Demo
-      code={propsToCode(level)}
-      propControls={<HeadingPropControls level={level} setLevel={setLevel} />}
+      code={propsToCode(level, isTruncated, value)}
+      propControls={
+        <HeadingPropControls
+          level={level}
+          setLevel={setLevel}
+          isTruncated={isTruncated}
+          setIsTruncated={setIsTruncated}
+          value={value}
+          setValue={setValue}
+        />
+      }
     >
-      <Heading level={level}>Heading</Heading>
+      <Heading width="30vw" level={level} isTruncated={isTruncated}>
+        {isTruncated}
```

Comment:
Why is `isTruncated` included in the children of `Heading`?

Reply:
Removed...